package.json

```diff
@@ -189,13 +189,13 @@
     "docs": "node ./site/scripts/docs.mjs",
     "watch": "c8 --reporter=lcov ava --watch",
     "site:build": "run-s site:build:*",
-    "site:build:install": "cd site && npm ci --no-audit",
+    "site:build:install": "cd site && npm ci --no-audit --legacy-peer-deps",
```

Comment:
I would prefer not to do this, but the alternative is to do https://github.com/netlify/cli/issues/304, as our site dependencies are quite old